### DISCUSSION TITLE
Export point de guidage avec le DXF avec la version 6.0

### DISCUSSION THREAD

**Discussion by christophe49250** [lairdubois/lairdubois-opencutlist-sketchup-extension#584] [Ideas]
Bonjour, 
J'ai vu la vidéo de la prochaine version de l'excellent "OpenCutlist 6.0" qui vas exporter des faces en 2D et 3D en : svg , dxf , obj ... 
Je suis très impatiens de voir la version définitive et bravo pour tout ce travail. 
J'utilise mes projets Sketchup pour des fabrications avec une CNC, j'exporte les profiles en DXF et ensuite je les re-travails avec un logiciel spécifique. Le problème est que l'exportation ne contiens pas les  **Points de guidage de sketchup** Il sont très utile pour les programmations de perçage et le centrage de poche.
Est t'il possible d'intégrer cette fonction dans la prochaine version ? Est ce possible ?
Merci bonne soirée

**Comment by bbeaulant** [comment]
Bonjour @christophe49250,

C'est pas impossible mais ce n'est pas prévu pour le moment. Surtout parce que le SVG n'apporte pas cette conception de "point" et dans un premier temps, je préfère que les deux formats restent au maximum avec les mêmes infos.

Quel logiciel utilises-tu pour modifier les DXF ?

**Reply by christophe49250** [reply]
Bonjour,
Merci pour cette réponse et je comprend. J'espère toujours voir cette option arriver sur une prochaine version!
Sinon j'utilise "pCon.planner" gratuit pour lire et modifier mes DXF et surtout le logiciel  "Maestro" spécifique à ma CNC pro du lycée.
J'adore faire des projets sympas pour des BAC PRO TMA et des CAP MEN.

**Reply by bbeaulant** [reply]
Je ne vais pas pouvoir tester ses logiciels. Mais les cercles étant transformé, leurs centre peuvent être retrouvés par le lecteur DXF.

**Reply by christophe49250** [reply]
C'est juste une manipulation de moins à réaliser et donc plus rapide. Parfois le cercle en exportation de SketchUp en DXF est éclaté et donc il n'est pas possible de retrouvé le centre...

**Reply by bbeaulant** [reply]
J'ai ajouté un algo qui recompose les cercles à partir des segments pour OCL 6.
Dans les DXF exportés par OCL, il pourra donc y avoir ces centres.

Exemple de cette pièce de SketchUp exportée par OCL en DXF
<img width="817" alt="Capture d’écran 2023-12-01 à 16 12 43" src="https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/156056/dd4592c2-5486-49e6-b5c5-8408b07df790">

Et importée dans Fusion360. Il trouve les centres tout seul.
<img width="725" alt="Capture d’écran 2023-12-01 à 16 11 59" src="https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/156056/0fa86286-0e08-4f73-96ca-46cdc0eace9d">

**Reply by christophe49250** [reply]
Ok je pense que ça devrais suffire. j'ai hâte de tester OCL 6.
Merci bonne fin de journée
